voorhoede/head-start
# feat: create custom 404 pages

# Changes

- Add custom 404 pages.
- I've created a postbuild script that rewrites e.g. `/en/404/index.html` to `/en/404.html` so that Cloudflare will pick these up when you navigate to a 404 within the scope of a locale.

# Associated issue

Resolves #9

# How to test

1. Open preview link
2. Navigate to a non existing page within the scope of a locale, e.g. /en/aapje
3. Verify that a not found page is rendered

# Checklist

~- [ ] I have added/updated tests to prove that my feature works (if not possible please explain why)~
~- [ ] I have made changes to the README and if the change affects the project setup (npm commands changed, new service added, environmental variable added)~
~- [ ] I have added a decision log entry if the change affects the architecture or changes a significant technology~
